support-files/mariadb.service.in

```diff
@@ -57,8 +57,8 @@ PrivateDevices=true
 # Prevent accessing /home, /root and /run/user
 ProtectHome=true
 
-# Execute pre and post scripts as root, otherwise it does it as User=
-PermissionsStartOnly=true
```

Comment:
Removing PermissionStartOnly will require #1105 and an alternative to `SYSTEMD_EXECSTARTPOST="ExecStartPost=/etc/mysql/debian-start"` (aka MDEV-12484).